src/domain/community/contributor/ContributorTooltip/ContributorTooltip.tsx

```
@@ -0,0 +1,82 @@
+  children,
+  ...props
+}: ContributorTooltipProps) => {
+  const [fetchUserData, { data, loading }] = useContributorTooltipLazyQuery();
+
+  const onOpenTooltipShow = () => {
+    if (!data && !loading && !override) {
+      switch (contributorType) {
+        case RoleSetContributorType.User:
+          fetchUserData({ variables: { userId: contributorId, includeUser: true } });
+          break;
+        case RoleSetContributorType.Organization: // TODO: Pending implementation
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)